src/odin/signs.cc

```diff
@@ -126,12 +127,35 @@ std::vector<Sign>* Signs::mutable_guide_toward_list() {
 
 const std::string Signs::GetGuideTowardString(uint32_t max_count,
                                               bool limit_by_consecutive_count,
-                                              std::string delim,
+                                              const std::string& delim,
                                               const VerbalTextFormatter* verbal_formatter) const {
-  return ListToString(guide_toward_list_, max_count, limit_by_consecutive_count, std::move(delim),
+  return ListToString(guide_toward_list_, max_count, limit_by_consecutive_count, delim,
                       verbal_formatter);
 }
 
+const std::string Signs::GetGuideString(uint32_t max_count,
+                                        bool limit_by_consecutive_count,
+                                        const std::string& delim,
+                                        const VerbalTextFormatter* verbal_formatter) const {
+  std::string guide_string;
+  if (HasGuideBranch() && HasGuideToward() && ((max_count == 0) || (max_count > 1))) {
```

Comment:
max_count == 0 means no limit
so this is checking if there is no limit or greater than one
we want to split the max_count between branch and toward

Reply:
I get the split, but not the need for the check (`((max_count == 0) || (max_count > 1))` is always true). Also `std::round` isn't doing anything, since we're using integer division which will return an integer.

Reply:
> A slightly more concise version of what this function is doing:

This is different functionality

Reply:
> I get the split, but not the need for the check (`((max_count == 0) || (max_count > 1))` is always true).

It is not true if max_count == 1 
I was trying to be explicit about either unlimited count or greater than one

Reply:
> Also `std::round` isn't doing anything, since we're using integer division which will return an integer.

@mandeepsandhu good catch - I will fix, thanks